presentation/components/src/test/instance-filter-builder/PresentationInstanceFilterBuilder.test.tsx

```diff
@@ -94,11 +141,55 @@ describe("PresentationInstanceFilter", () => {
     // wait until operator is selected
     await waitFor(() => getByText(getPropertyFilterOperatorLabel(PropertyFilterRuleOperator.IsNotNull)));
 
-    expect(spy).to.be.calledOnceWith({
-      field: propertiesField,
-      operator: PropertyFilterRuleOperator.IsNotNull,
-      value: undefined,
-    });
+    expect(spy).to.be.calledWith({
+      filter: {
+        field: propertiesField,
+        operator: PropertyFilterRuleOperator.IsNotNull,
+        value: undefined,
+      },
+      usedClasses: [classInfo],
+    }
+    );
+  });
+
+  it("invokes 'onInstanceFilterChanged' with filter", async () => {
```

Comment:
Does this test really need to test that filter is updated when property is selected? It looks like previous test does the same. Maybe in this it would be enough to check that PresentationInstanceFilterBuilder was renderer with correct initial state (there are 2 rules and correct properties are selected)?